docs/concepts/05_credentials/05_verification.md

```diff
@@ -3,68 +3,97 @@ id: verification
+This increases the privacy of the Claimer since they only need to show attributes required in the specific context.
+
+:::info
+
+For a detailed developer-oriented guide to KILT presentation creation, read the [presentation creation cookbook section](../../develop/01_sdk/02_cookbook/04_claiming/04_presentation_creation.md).
+
+:::
+
+## Verifying a presentation
 
 The Verifier receives the presentation from the Claimer, re-calculates the root hash of the credential from which the presentation was generated, and queries the KILT blockchain to obtain the associated attestation information, including the revocation status of the credential.
 
-If the Claimer tampered with the credential, the re-calculated root hash will not match any attestation on the chain.
-On the other hand, if an attestation with the calculated hash can be found on the chain and has not been revoked, the credential is valid.
+If the Claimer tampered with the credential, the re-calculated root hash won't match any attestation on the chain.
+On the other hand, if the Verifier can find an attestation with the calculated hash on the chain and hasn't been revoked, the credential is valid.
 
-However, this does not give the Verifier the guarantee that the Claimer is the rightful/legitimate owner of the credential presented.
-That problem is addressed in the next section.
+However, this doesn't give the Verifier the guarantee that the Claimer is the rightful/legitimate owner of the credential presented.
 
-### Verifying the Owner of the Presented Credential
+### Verifying the owner of the presented credential
 
 When issued, a credential is linked to the KILT decentralized identifier (DID) of the original Claimer.
-The DID can be resolved to the public key of the Claimer according to the [KILT DID specification](https://github.com/KILTprotocol/spec-kilt-did).
+The Verifier can resolve the DID to the public key of the Claimer according to the [KILT DID specification](https://github.com/KILTprotocol/spec-kilt-did).
+
+The Verifier assumes that the private key for the DID public key is only known to the owner of the credential, and isn't shared across users.
+Therefore, when requesting the Claimer to generate a presentation, the Verifier challenges the Claimer to sign a nonce (a random number used once) that the Verifier sends together with their request.
 
-The Verifier assumes that the private key for the DID public key is only known to the owner of the credential, and is not shared across users.
-Therefore, when requesting the Claimer to generate a presentation, the Verifier challenges the Claimer to sign a nonce (a random number that is used once) that the Verifier sends together with their request.
+If the Claimer can sign both the nonce and the presentation with the private key that only the credential's owner should have knowledge of, the Verifier won't be sure that the Claimer is the legitimate owner of the credential.
```

Comment:
```suggestion
If the Claimer can sign both the nonce and the presentation with the private key that only the credential's owner should have knowledge of, the Verifier can be sure that the Claimer is the legitimate owner of the credential.
```

Reply:
It is only one signature inside the presentation. The credential and the nonce are part of the input for it.  

I don't know if this is so important though.

Reply:
Example Credential Presentation:

```
{
   "claim": {
      "cTypeHash": "0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914b9ac",
      "contents": {
         "Email": "refij28030@in2reach.com"
      },
      "owner": "did:kilt:4tFve5tV8dA8hQ74Q2jFYB8sep7HS2Mi4Zdm9DFa25rtiu5E"
   },
   "claimHashes": [
      "0x68176b55844c0645a0bd778cafb67185581cdb32106b89a8691d29b8402e37cd",
      "0xf8b622c16489ddf7bd601bbbbe69e3e5e865341ac001d08816a89c977fa2a4e7"
   ],
   "claimNonceMap": {
      "0xec2a9c4ed9575eb5024d93d9c7d2a08430d9a642e6da067386d091f2b419c2ac": "df741c65-ba1d-4b21-9b61-0fae5e2caa45",
      "0x702ce98117a4772b27547e61b6ab035da2c2947852c933c1888e0c40bce47f75": "8ae79e6a-ff2f-4a8c-8061-b2d33243d006"
   },
   "delegationId": null,
   "legitimations": [],
   "rootHash": "0x87d4140cf379e201746567bb4c1aad025965255362d66ea816104219b3e188e4",
   "claimerSignature": {
      "signature": "0xbe3f49755f15e23bbad60c845cfc9ec735e3663728cfb0c588186e4d34272b27cb23ff299180da2eaa966b03dcde2273f29d610df46a08c530b6418cf45a2683",
      "keyUri": "did:kilt:4tFve5tV8dA8hQ74Q2jFYB8sep7HS2Mi4Zdm9DFa25rtiu5E#0x74906624b68b9c2231d26e0df563499e3ef411982ec4da9f604743447d48adb8",
      "challenge": "0x404944876708af61a71e885954eda5ccea39b2ceae873168c96617d23f74bc40"
   }
}
```